src/test/java/racingcar/CarTest.java

```diff
@@ -0,0 +1,59 @@
+package racingcar;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingcar.controller.InputHandler;
+import racingcar.model.Participants;
+import racingcar.model.car.Car;
+import racingcar.service.CarSaveService;
+
+public class CarTest {
+
+    @Test
+    @DisplayName("이름은 대소문자 구분없이 1~5글자의 영문이다.")
+    void 자동차_테스트(){
+        Car.of(0,"j");
+        Car.of(0,"jo");
+        Car.of(0,"joh");
+        Car.of(0,"john");
+        Car.of(0,"joHnN");
+    }
+    @Test
+    @DisplayName("공백, 띄어쓰기, 지정되지 않은범위의 이름은 예외가 발생한다.")
+    void 자동차_테스트2(){
+        Assertions.assertThatThrownBy(()->
+                Car.of(0,"")).isInstanceOf(IllegalArgumentException.class);
+
+        Assertions.assertThatThrownBy(()->
+                Car.of(0,"a b")).isInstanceOf(IllegalArgumentException.class);
+
+        Assertions.assertThatThrownBy(()->
+                Car.of(0,"johnnn")).isInstanceOf(IllegalArgumentException.class);
+    }
```

Comment:
다른 분들 코드를 리뷰하다 알게 된 건데, 이런식으로 [@ValueSource](https://junit.org/junit5/docs/current/api/org.junit.jupiter.params/org/junit/jupiter/params/provider/ValueSource.html) 를 활용하면 코드 중복을 해결할 수 있을 것 같아요 !

```suggestion
    @ParameterizedTest
    @DisplayName("공백, 띄어쓰기, 지정되지 않은범위의 이름은 예외가 발생한다.")
    @ValueSource(strings = {"", "a b", "johnnn"})
    void 자동차_테스트2(String invalidName){
        Assertions.assertThatThrownBy(()->
                Car.of(0, invalidName)).isInstanceOf(IllegalArgumentException.class);
    }
```

Reply:
코드까지 직접 쳐주셨네요 감사합니다 :) 맞습니다. 좋은 애노테이션인 것 같아요!

Reply:
저도 배워갑니다!